Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM registry.gitlab.com/pages/hugo/hugo_extended:latest
+FROM klakegg/hugo:0.101.0-ext-alpine
```

Comment:
```suggestion
FROM registry.gitlab.com/pages/hugo/hugo_extended:latest
```

Reply:
Is there a poll feature here? Because I still stand by my original argument that Klackett is the image the Hugo project themselves recommended and comes from Docker Hub (maybe more reliable than gitlab registry? I don't know).

Perhaps we can just use reactions. 

- [ ] Thumbs up for Hugo project recommendation (Klackett)
- [ ] Smiley face for gitlab image

Reply:
Sounds good, we can try and do the emoji reactions for voting. Did you see my comment that they actually do recommend the gitlab one as well?

> Also, the hugo docs do recommend using that image, but it's just on the Gitlab deployment page: https://gohugo.io/hosting-and-deployment/hosting-on-gitlab/

---

> maybe more reliable than gitlab registry?

Ya, I don't know about this one, I'd imagine Gitlab's registry would have to be pretty rock solid, since they tout themselves as the "The One DevOps Platform" for businesses. I'd argue they'd probably have to be more resilient, because of how many businesses they probably support compared to docker is still trying to find a way to become profitable. Plus the docker registry has their [rate limiting now](https://container-registry.com/posts/overcome-docker-hub-rate-limit/).

Reply:
I have reverted the image to `registry.gitlab.com/pages/hugo/hugo_extended:latest` based on the voting results (1:4) in favor.